test/await-use.test.js

```diff
@@ -133,9 +132,9 @@ test('await use - error handling, async throw, nested', async (t) => {
     await app.use(async function b (f, opts) {
       throw Error('kaboom')
     })
-  }, Error('kaboom')), 'b')
+  }), Error('kaboom'), 'b')
 
-  t.rejects(() => app.ready(), Error('kaboom'))
+  t.assert.rejects(() => app.ready(), Error('kaboom'))
```

Comment:
Not sure if we should await the rejects here - because tap was checking the plan, but node:test does not
```suggestion
  await t.assert.rejects(() => app.ready(), Error('kaboom'))
```